gr-fec/lib/fec_mtrx_impl.cc

```diff
@@ -148,39 +142,43 @@ void write_matrix_to_file(const std::string filename, matrix_sptr M)
 
 matrix_sptr generate_G_transpose(matrix_sptr H_obj)
 {
-    unsigned int k = H_obj->size1;
-    unsigned int n = H_obj->size2;
-    unsigned int row_index, col_index;
+    const unsigned int k = H_obj->size1;
+    const unsigned int n = H_obj->size2;
```

Comment:
I'm not familiar with GSL matrix objects. Before, `H_obj` was a `struct` defined by GNU Radio which contained `size1` and `size2` fields, so this code makes sense. I guess that a `gsl_matrix_uchar` also contains these fields under the hood, but is the user supposed to access the fields, or is it better to treat the struct as an opaque object and use getter functions?